scripts/live_test/CLITest.yml

```diff
@@ -0,0 +1,259 @@
+        azdev setup -c azure-cli
+        # az login --service-principal --username $(APP_ID) --password $(PASSWORD) --tenant $(TENANT_ID)
+        az login -u azureclitest@azuresdkteam.onmicrosoft.com -p $(AZURECLITESTPASSWORD)
+        az account set -s 0b1f6471-1bf0-4dda-aec3-cb9272f09590
+        # Sequential
+        azdev test $(Target) $(USER_LIVE) --mark serial --xml-path test_results.sequential.xml --no-exitfirst -a "--json-report --json-report-summary --json-report-file=$(Target).report.sequential.json --html=$(Target).report.sequential.html --self-contained-html --reruns 3 -s"
+        # Parallel
+        azdev test $(Target) $(USER_LIVE) --mark "not serial" --xml-path test_results.parallel.xml --no-exitfirst -a "-n $(USER_PARALLELISM) --json-report --json-report-summary --json-report-file=$(Target).report.parallel.json --html=$(Target).report.parallel.html --self-contained-html --reruns 3 -s"
+        pwd
+        ls
+      condition: eq(variables.Match, '1')
+    - task: PublishTestResults@2
+      condition: eq(variables.Match, '1')
+      inputs:
+        testResultsFiles: '/home/vsts/work/1/test_results.*.xml'
+        testRunTitle: 'Azure CLI live test results of $(Target)'
+        mergeTestResults: true
+        # testResultsFormat: 'JUnit'
+    - task: PublishBuildArtifacts@1
+      condition: eq(variables.Match, '1')
+      inputs:
+        PathtoPublish: '/home/vsts/work/1/$(Target).report.sequential.json'
+        ArtifactName: '$(Target).report.sequential.json'
+        publishLocation: 'Container'
+    - task: PublishBuildArtifacts@1
+      condition: eq(variables.Match, '1')
+      inputs:
+        PathtoPublish: '/home/vsts/work/1/$(Target).report.sequential.html'
+        ArtifactName: '$(Target).report.sequential.html'
+        publishLocation: 'Container'
+    - task: PublishBuildArtifacts@1
+      condition: eq(variables.Match, '1')
+      inputs:
+        PathtoPublish: '/home/vsts/work/1/$(Target).report.parallel.json'
+        ArtifactName: '$(Target).report.parallel.json'
+        publishLocation: 'Container'
+    - task: PublishBuildArtifacts@1
+      condition: eq(variables.Match, '1')
+      inputs:
+        PathtoPublish: '/home/vsts/work/1/$(Target).report.parallel.html'
+        ArtifactName: '$(Target).report.parallel.html'
+        publishLocation: 'Container'
+
+- job: SendEmail
+  displayName: Send email
+  dependsOn: LiveTestPython38
+  condition: succeededOrFailed()
+  steps:
+    - task: UsePythonVersion@0
+      inputs:
+        versionSpec: '3.8'
+        addToPath: true
+        architecture: 'x64'
+    - task: DownloadBuildArtifacts@0
+      inputs:
+        buildType: 'current'
+        downloadType: 'specific'
+        downloadPath: '$(System.ArtifactsDirectory)'
+    - bash: |
+        echo $(System.ArtifactsDirectory)
+        ls $(System.ArtifactsDirectory)
+        pwd
+        commit_id=`git ls-remote https://github.com/Azure/azure-cli.git HEAD`
+        pip install sendgrid
```

Comment:
Using sendgrid to send emial obey our Microsoft's policy? I am not sure.

Reply:
Neither am I.